config-templates/config.php

```diff
@@ -861,6 +861,37 @@
      */
     'production' => true,
 
+    /*
+     * Set this option to adjust basic caching directives sent by module.php.
+     *
+     * Defaults to ['public' => true, 'max_age' => 86400]
+     */
+    // 'module.php.cache' => ['public' => true, 'max_age' => 86400],
+
+    /*
+     * Set this option to adjust expiration timestamp sent by module.php.
+     * The value (in seconds) is added to current time.
+     * If set to null, the Expiration header is not sent at all.
+     *
+     * Defaults to 10 * 60 (ten minutes).
+     */
+    // 'module.php.expires' => 10 * 60,
+
+    /*
+     * Set this option to make module.php automatically send Etag.
+     *
+     * Defaults to false.
+     */
+    // 'module.php.etag' => false,
+
+    /*
+     * Set this option to adjust more caching directives sent by module.php,
+     * for example must-revalidate. The option takes an array
+     * where keys are the caching directive names and values are the corresponding values.
+     *
+     * Defaults to [] (empty array).
+     */
+    // 'module.php.directives' => [],
```

Comment:
This is very confusing. I think we need to either move every directive we want to support to the general `caching` configuration group, or remove it. My main problem with this is that it forces you to know Symfony in order to configure it. Flexibility is nice, but I wouldn't depend on any Symfony-specific code.